# This msg file specifies all arrangement states related to
# the object specified

int32 obj_idx
confined_space_rearrangement/ArrState[] invalid_arr_states